hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java

```diff
@@ -309,7 +314,19 @@ public int getDepth() {
         final Map<String, Object> config = objectMapper.convertValue(adapter.getConfig(), new TypeReference<>() {
         });
 
-        protocolAdapterManager.updateAdapterConfig(adapterId, config);
+
+        // TODO we need to extract the mappings from the adapter object
```

Comment:
What are we doing with these TODOs?
Another PR?

Reply:
we fix it in this PR if any possible